src/custom-resource-loader_test.js

```diff
@@ -0,0 +1,264 @@
+// @flow
```

Comment:
A whole new file of tests to verify our custom loader is working the way we expect.